docs/application-connector/docs/004-architecture-application-connector.md

```diff
@@ -45,7 +45,7 @@ The controller listens for creating or deleting the RemoteEnvironment Custom Res
 
 ## Proxy Service
 
-The Proxy Service is an intermediary component between a lambda function or a service and an external API registered with the Metadata Service. It acts as a proxy and can acquire OAuth tokens.
+The Proxy Service is an intermediary component between a lambda function or a service and an external API registered with the Metadata Service. It can call services secured with the [Basic Authentication](https://tools.ietf.org/html/rfc7617) mechanism, and can acquire OAuth tokens and call OAuth-secured APIs.  
```

Comment:
```suggestion
The Proxy Service is an intermediary component between a lambda function or a service and an external API registered with the Metadata Service. It can call services secured with the [Basic Authentication](https://tools.ietf.org/html/rfc7617) mechanism, acquire OAuth tokens, and call OAuth-secured APIs.  
```